package.json

```diff
@@ -69,7 +70,10 @@
     "eslint-plugin-testing-library": "^4.12.3",
     "jest": "^27.2.1",
     "jest-fetch-mock": "^3.0.3",
+    "jest-puppeteer": "^5.0.4",
```

Comment:
I just noticed that this was in here (added by me when I first set up jest stuff).  We can actually remove this (and the puppeteer module), since it's only relevant to the unmerged jest feature tests in https://github.com/cul/ldpd-hyacinth/tree/experimental-jest-e2e-testing.

Reply:
when I squashed I included `yarn remove` on the two puppeteer-related dependencies, thanks for catching this!